apps/meteor/tests/unit/app/livechat/server/lib/parseTranscriptRequest.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { expect } from 'chai';
+import p from 'proxyquire';
+import sinon from 'sinon';
+
+const modelsMock = {
+	Users: {
+		findOneById: sinon.stub(),
+	},
+	LivechatVisitors: {
+		findOneById: sinon.stub(),
+	},
+};
+
+const settingsGetMock = {
+	get: sinon.stub(),
+};
+
+const { parseTranscriptRequest } = p.noCallThru().load('../../../../../../app/livechat/server/lib/parseTranscriptRequest', {
+	'@rocket.chat/models': modelsMock,
+	'../../../settings/server': { settings: settingsGetMock },
+});
+
+describe('parseTranscriptRequest', () => {
+	beforeEach(() => {
+		settingsGetMock.get.reset();
+		modelsMock.Users.findOneById.reset();
+		modelsMock.LivechatVisitors.findOneById.reset();
+	});
+
+	it('should return options when Livechat_enable_transcript setting is true', async () => {
```

Comment:
I think it would be better to have the meaning for that also, like: "should return the object allowing the request etc etc". WDYT?

Reply:
The same for all other cases